Yoast/wordpress-seo
# Use refresh instead of runAnalyzer to avoid delays after typing

## Summary

This PR can be summarized in the following changelog entry:

* N/A

The analysis was run on every keystroke, which caused delays when typing in the snippet editor.

## Relevant technical choices:

* app.runAnalyzer is not debounced, while app.refresh is (and app.refresh calls app.runAnalyzer). Using app.refresh will avoid delays after typing, because the analysis will not longer be run after every keystroke.

## Test instructions

This PR can be tested by following these steps:

* Open the inspector, click performance. Record when you are typing. Check that the runAnalyzer isn't running on every keystroke. 

## Quality assurance
